roles/ansible-runner/tasks/runner.yml

```diff
@@ -33,6 +33,12 @@
   secret_file_path: "{{ ansible_runner_secrets_path }}"
   secret_file_user: "{{ item.user }}"
 
+- name: Disable cron job
+  copy:
```

Comment:
(nit) I would prefer to use the file module with `state: touch` instead of the copy module.

(-1) Let's also remove the file when the runner is not disabled